src/containers/tutor-home-page/subjects-step/SubjectsStep.jsx

```diff
@@ -44,16 +48,33 @@ const SubjectsStep = ({ btnsBox }) => {
         console.error('Error fetching subjects:', error)
       }
     }
-
     fetchSubjects()
   }, [selectedCategory])
-
+  const dataChipList = {
+    items: [...selectedSubjects],
+    defaultQuantity: 2,
+    handleChipDelete: (deletedItem) =>
+      setSelectedSubjects(
+        [...selectedSubjects].filter((item) => item !== deletedItem)
```

Comment:
`filter` returns a new array. We do not need `...` here.
```suggestion
        selectedSubjects.filter((item) => item !== deletedItem)
```

Reply:
i fix it